### PR TITLE
Cleanup code added for CSI support

### PR DESCRIPTION
**Pull request by shivamerla**
Signed-off-by: Shiva Krishna, Merla <shivakrishna.merla@hpe.com>

### CODE REVIEW

File: pkg/provisioner/provisioner.go [hunk 1]
```diff
@@ -101,8 +101,6 @@ type Provisioner struct {
 	deleteCommandChains     uint32
 	parkedCommands          uint32
 	debug                   bool
```
Comment (raunakkumar) [inline pkg/provisioner/provisioner.go:102]: nit: vaStore constant can be removed

**Review by raunakkumar — APPROVED** [review]
lgtm. 
also newCsiPersistentVolume() can be removed from volume.go

File: pkg/provisioner/provisioner.go (hunk 1 above)
Comment (shivamerla) [reply pkg/provisioner/provisioner.go:102]: done..

**Comment by shivamerla** [comment]
> lgtm.
> also newCsiPersistentVolume() can be removed from volume.go

done

**Pull request merged**